test/Confluent.SchemaRegistry.IntegrationTests/Tests/ProtobufWithReferences.cs

```diff
@@ -25,6 +25,11 @@ public static partial class Tests
         [Theory, MemberData(nameof(SchemaRegistryParameters))]
```

Comment:
This test looks to be failing always. Can you add a `FIXME` comment here to fix this test?
And let's comment this line `cd test/Confluent.SchemaRegistry.IntegrationTests && dotnet test -l "console;verbosity=normal" && cd ../..` in the semaphore.yml.

Reply:
Done !